third_party/ijar/test/zip_test.sh

```diff
@@ -188,4 +188,22 @@ function test_zipper_specify_path() {
       || fail "Unzip after zipper output is not expected"
 }
 
+function test_zipper_permissions() {
+  mkdir -p ${TEST_TMPDIR}/files
```

Comment:
${TEST_TMPDIR} may be not be a dedicated temp directory, it may be `/tmp`, and the danger of name collision with an existing file called "files" is real.

I recommend using a more unique name, and creating mock files/directories in the current directory.
Example:
https://github.com/bazelbuild/bazel/blob/2fda17d46088f2bb07b049fa35759002ea1dd5db/src/test/shell/integration/progress_reporting_test.sh#L100-L101

Reply:
https://docs.bazel.build/versions/master/test-encyclopedia.html#test-interaction-with-the-filesystem indicates that the `TEST_TMPDIR` will be empty prior to running the test.

I've namespaced this test's test directory using the function name to avoid collisions with other tests however.